scripts/test_urls.vim

```diff
@@ -0,0 +1,68 @@
+" Test for URLs in help documents.
+"
+" Opens a new window with all found URLS followed by return code from curl
+" (anything other than 0 means unreachable)
+"
+" Written by Christian Brabandt.
+
+func Test_check_URLs()
```

Comment:
This is named like it was intended to be run as a test in `src/testdir`, but maybe they decided not to .